trl/trainer/dpo_trainer.py

```diff
@@ -223,6 +226,7 @@ def __init__(
         self.padding_value = padding_value
 
         self.beta = beta
+        self.loss_type = loss_type
```

Comment:
I think we should raise an error if someone provided the wrong loss string. I guess currently they would get a "Losses referenced before definition" kind of error.

Otherwise looks good to me!

Reply:
ok fixing